frontend/public/components/cluster-service-class.tsx

```diff
@@ -46,7 +47,7 @@ const ClusterServiceClassListRow: React.SFC<ClusterServiceClassRowProps> = ({obj
       {serviceClass.spec.externalName}
     </div>
     <div className="col-sm-3 hidden-xs">
-      {serviceClass.spec.clusterServiceBrokerName}
+      <ResourceLink kind="ClusterServiceBroker" name={serviceClass.spec.clusterServiceBrokerName} title={serviceClass.spec.clusterServiceBrokerName} />
```

Comment:
Remove `title` attribute here... It's just repeating what's already displayed and doesn't add anything since we don't truncate.